susemanager/bin/mgr-setup

```diff
@@ -199,75 +199,108 @@ cleanup_hostname() {
 
 exists_db() {
     PGNAME=$1
-    EXISTS=$(runuser - postgres -c 'psql -t -c "select datname from pg_database where datname='"'$PGNAME'"';"')
+    if [ $LOCAL_DB = 1 ]; then
+        EXISTS=$(runuser - postgres -c 'psql -t -c "select datname from pg_database where datname='"'$PGNAME'"';"')
+    else
+        EXISTS=$(echo "select datname from pg_database where datname='$PGNAME';" | (export PGPASSWORD=$EXTERNAL_DB_PASS; exec psql -t -U postgres -h $MANAGER_DB_HOST))
```

Comment:
I think postgres user should be replace by $EXTERNAL_DB_USER other it's force the RDS to be created with postgres user and $EXTERNAL_DB_USER = postgres .

Reply:
good catch @maximenoel8  I thought in some places I did this but let me make sure and push an update. Thank you.

Reply:
fixed with my latest push